builtin/providers/aws/resource_aws_redshift_cluster.go

```diff
@@ -590,3 +625,12 @@ func validateRedshiftClusterMasterUsername(v interface{}, k string) (ws []string
 	}
 	return
 }
+
+func buildRedshiftARN(identifier, accountid, region string) (string, error) {
+	if accountid == "" {
+		return "", fmt.Errorf("[ERROR] No AccountId found")
```

Comment:
I'd personally make this error message more specific, also if it's already an error message, it doesn't need to contain `[ERROR]`.

How about `Unable to construct cluster ARN because of missing AWS Account ID`?